server/controllers/image.js

```diff
@@ -0,0 +1,16 @@
+const asyncHandler = require("express-async-handler");
+
+const uploadFile = require("../utils/s3");
+
+// @route POST /users
+// @desc Search for users
+// @access Private
```

Comment:
Good documentation

Reply:
Noted, my bad lol; changed to better reflect method.